multicluster/controllers/multicluster/resourceexport_controller.go

```diff
@@ -132,13 +132,13 @@ func (r *ResourceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 	if createResImport {
 		if err = r.Client.Create(ctx, resImport, &client.CreateOptions{}); err != nil {
-			klog.ErrorS(err, "failed to create ResourceImport", "resourceimport", resImportName.String())
+			klog.ErrorS(err, "Failed to create ResourceImport", "resourceimport", resImportName.String())
 			return ctrl.Result{}, err
 		}
 		r.updateResourceExportStatus(&resExport, succeed)
-		klog.InfoS("create ResourceImport successfully", "resourceimport", resImportName.String())
+		klog.V(2).InfoS("ResourceImport is created successfully", "resourceimport", resImportName.String())
 	} else if changed {
-		klog.InfoS("update ResourceImport for ResoureExport", "resourceimport", resImportName.String(), "resourceexport", req.NamespacedName)
+		klog.InfoS("Updating ResourceImport for ResoureExport", "resourceimport", resImportName.String(), "resourceexport", req.NamespacedName)
```

Comment:
why updating and creating use different level logs?

Reply:
I was trying to reduce logs, let me make them consistent.

Reply:
fixed.